client/reader/conversations/intro.jsx

```diff
@@ -1,106 +1,59 @@
-		if ( props.hasUsedConversations !== true ) {
-			this.props.recordReaderTracksEvent( 'calypso_reader_conversations_intro_render' );
+const ConversationsIntro = ( { isInternal = false, hasUsedConversations, dismiss } ) => {
+	const translate = useTranslate();
+	useEffect( () => {
+		if ( hasUsedConversations !== true ) {
+			recordReaderTracksEvent( 'calypso_reader_conversations_intro_render' );
 		}
-	};
+	}, [ hasUsedConversations ] );
 
-	dismiss = () => {
-		this.props.recordReaderTracksEvent( 'calypso_reader_conversations_intro_dismiss' );
-		this.props.dismiss( this.props.isInternal );
+	const onClose = () => {
+		recordReaderTracksEvent( 'calypso_reader_conversations_intro_dismiss' );
+		dismiss( isInternal );
 	};
 
-	render() {
-		const { hasUsedConversations, translate, isInternal } = this.props;
-
-		if ( hasUsedConversations ) {
-			return null;
-		}
-
-		return (
-			<header className="conversations__intro">
-				<QueryPreferences />
-				<div className="conversations__intro-header">
-					<div className="conversations__intro-copy">
-						<span>
-							{ isInternal
-								? translate(
-										`{{strong}}Welcome to A8C Conversations.{{/strong}} ` +
-											`Automattic P2 posts you've written, followed, or commented on will appear here when they have new comments. ` +
-											`Posts with the most recent comments appear on top. ` +
-											`{{a}}More info.{{/a}}`,
-										{
-											components: {
-												strong: <strong />,
-												a: <a href="http://wp.me/p5PDj3-44u" />,
-											},
-										}
-								  )
-								: translate(
-										`{{strong}}Welcome to Conversations{{/strong}} ` +
-											`WordPress posts you've written, followed, or commented on will appear here when they have new comments. ` +
-											`Posts with the most recent comments appear on top.`,
-										{
-											components: {
-												strong: <strong />,
-											},
-										}
-								  ) }
-						</span>
-					</div>
-
-					<button
-						className="conversations__intro-close"
-						onClick={ this.dismiss }
-						title={ translate( 'Close' ) }
-						aria-label={ translate( 'Close' ) }
-					>
-						<Gridicon
-							icon="cross-circle"
-							className="conversations__intro-close-icon"
-							title={ translate( 'Close' ) }
-						/>
-					</button>
-				</div>
-			</header>
-		);
+	if ( hasUsedConversations ) {
+		return null;
 	}
-}
+	return (
+		<>
+			<QueryPreferences />
+			<NoticeBanner
+				level="info"
+				title={ translate( 'Welcome to A8C Conversations.' ) }
+				onClose={ onClose }
+			>
+				<span>
+					{ isInternal
+						? translate(
+								`Automattic P2 posts you've written, followed, or commented on will appear here when they have new comments. ` +
+									`Posts with the most recent comments appear on top. ` +
+									`{{a}}More info.{{/a}}`,
+								{
+									components: {
+										a: <a href="http://wp.me/p5PDj3-44u" />,
```

Comment:
I think this should be an external link. Currently, it opens in the same window. Feel free to use `<ExternalLink />`.